src/main/windows.ts

```diff
@@ -50,7 +57,9 @@ export function createMainWindow(): Electron.BrowserWindow {
   console.log(`Creating main window`);
   let browserWindow: BrowserWindow | null;
   browserWindow = new BrowserWindow(getMainWindowOptions());
-  browserWindow.loadFile('./dist/static/index.html');
+  browserWindow.loadURL(
+    !!process.env.JEST ? './dist/static/index.html' : MAIN_WINDOW_WEBPACK_ENTRY,
```

Comment:
I think this should actually be `./.webpack/renderer/main_window/index.html`... I don't think the tests are actually making usage of it, but let's change it to that.

Same with `preload` above, let's change that to `./.webpack/renderer/main_window/preload.js`.